config/environments/development.rb

```diff
@@ -43,6 +39,9 @@
 
   config.cache_store = :dalli_store, nil, { value_max_bytes: 2.megabytes }
 
+  # Raises error for missing translations
+  config.action_view.raise_on_missing_translations = true
```

Comment:
oh wait sorry wrong place for the comment. I'm getting lost here. This PR is getting way too big.  This is set to false in prod right now.